definitions/tools/cnvnator.cwl

```diff
@@ -34,15 +34,15 @@ requirements:
           awk 'BEGIN { CHROM="" } { if ($1~"^>") CHROM=substr($1,2); print $0 > "FASTA_CHRS/"CHROM".fa" }' "$REFERENCE"
                     
           # extract read mapping from input bam(single sample)
-          cnvnator -root "${SAMPLE}.root" -tree "$BAM" -chrom "$CHROMOSOMES"
+          cnvnator -root "${SAMPLE}.root" -tree "$BAM" -chrom $CHROMOSOMES
```

Comment:
It might be better to have this script take the chromosomes at the end and
`shift 4` before using `"$@"` to pass the chromosome names along.  (This would entail no longer using the `itemSeparator` option in the CWL as well.)

Reply:
I created #792 and will work on updating then testing the cnvnator cwl tool as you suggested